Zend/Optimizer/zend_optimizer.c

```diff
@@ -914,6 +914,28 @@ zend_function *zend_optimizer_get_called_func(
 					}
 					return fbc;
 				}
+			} else if (opline->op1_type == IS_CONST
+					   && Z_TYPE_P(CRT_CONSTANT(opline->op1)) == IS_OBJECT
+					   && opline->op2_type == IS_CONST && Z_TYPE_P(CRT_CONSTANT(opline->op2)) == IS_STRING
+					   && op_array->scope
+					   && !(op_array->fn_flags & ZEND_ACC_TRAIT_CLONE)
+					   && !(op_array->scope->ce_flags & ZEND_ACC_TRAIT)) {
+				zend_string *method_name = Z_STR_P(CRT_CONSTANT(opline->op2) + 1);
+				zend_function *fbc = zend_hash_find_ptr(
```

Comment:
If you have the `op_array`, why do you need to look for the function? Note that they're the same thing, a `zend_function` is either internal (`zend_internal_function`) or user (`zend_op_array`).

Reply:
I had a brain fart here ^^ op_array refers to the caller here, ofc.